processor/configuration/src/test/java/com/redhat/service/smartevents/processor/validators/DefaultGatewayValidatorTest.java

```diff
@@ -48,6 +52,93 @@ void testSlackSource() {
         assertValidationIsValid(sourceWith(SlackSource.TYPE, params));
     }
 
+    @Test
+    void testSlackAction() {
+        Map<String, String> params = new HashMap<>();
+        params.put(SlackAction.CHANNEL_PARAM, "t");
+        assertValidationIsInvalid(actionWith(SlackAction.TYPE, params), List.of("$.slack_webhook_url: is missing but it is required"));
+
+        params = new HashMap<>();
```

Comment:
I am non expert in java garbage collecting, but shouldn't we just call `params.clear();` instead of new instance creation `params = new Hashmap<>();`